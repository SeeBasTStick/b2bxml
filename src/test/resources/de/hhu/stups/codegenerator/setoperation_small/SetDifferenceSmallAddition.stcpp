int main() {
    SetDifferenceSmall set;
    set.simulate();
    cout << set.getSet();
    return 0;
}