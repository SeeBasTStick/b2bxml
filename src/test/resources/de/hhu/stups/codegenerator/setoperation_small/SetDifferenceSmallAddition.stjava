public static void main(String[] args) {
    SetDifferenceSmall set = new SetDifferenceSmall();
    set.simulate();
    System.out.println(set.set);
}