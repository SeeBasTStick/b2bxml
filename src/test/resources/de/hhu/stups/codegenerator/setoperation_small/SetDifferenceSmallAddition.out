public static void main(String[] args) {
    SetComplementSmall set = new SetComplementSmall();
    set.simulate();
    System.out.println(set.set);
}